common/transforms/Ethiopic-MorseCode-Amharic.xml

```diff
@@ -0,0 +1,683 @@
+#
+# ዸ ↔ $ዸ $ግዕዝ ; # ETHIOPIC SYLLABLE DDA
+# ዿ ↔ $ዸ $ዘመደ_ራዕብ ; # ETHIOPIC SYLLABLE DDWA
+# ዹ ↔ $ዸ $ካዕብ ; # ETHIOPIC SYLLABLE DDU
+# ዼ ↔ $ዸ $ኃምስ ; # ETHIOPIC SYLLABLE DDEE
+# ዺ ↔ $ዸ $ሣልስ ; # ETHIOPIC SYLLABLE DDI
+# ዻ ↔ $ዸ $ራዕብ ; # ETHIOPIC SYLLABLE DDAA
+# ⶍ ↔ $ዸ $ሳምን ; # ETHIOPIC SYLLABLE DDOA
+# ዾ ↔ $ዸ $ሳብዕ ; # ETHIOPIC SYLLABLE DDO
+# ዽ ↔ $ዸ $ሳድስ ; # ETHIOPIC SYLLABLE DDE
+#
+# ⷘ ↔ $ገy $ግዕዝ ; # ETHIOPIC SYLLABLE GYA
+# ⷙ ↔ $ገy $ካዕብ ; # ETHIOPIC SYLLABLE GYU
+# ⷜ ↔ $ገy $ኃምስ ; # ETHIOPIC SYLLABLE GYEE
+# ⷚ ↔ $ገy $ሣልስ ; # ETHIOPIC SYLLABLE GYI
+# ⷛ ↔ $ገy $ራዕብ ; # ETHIOPIC SYLLABLE GYAA
+# ⷞ ↔ $ገy $ሳብዕ ; # ETHIOPIC SYLLABLE GYO
+# ⷝ ↔ $ገy $ሳድስ ; # ETHIOPIC SYLLABLE GYE
+
+
+########################################################################
+# Start of Punctuation Transformations
+########################################################################
+
+። ↔ '...... ' ;
+፤ ↔ '.-.-.- ' ;
+\" ↔ '.-..-. ' ;
+\? ↔ '..--..' ;
+\! ↔ '-.-.--' ;
+\$ ↔ '...-..- ' ;
+\[ → '-.--. ' ;
+\] → '-.--.- ' ;
+\( ↔ '-.--. ' ;
+\) ↔ '-.--.- ' ;
+\/ ↔ '-..-. ' ;
+\- ↔ '-....- ' ;
+\_ ↔ '..--.- ' ;
+\+ → '.-.-. ' ;
+\= → '-...- ' ;
+\@ ↔ '.--.-. ' ;
+\& ↔ '.-... ' ;
+\. → '.-.-.- ' ;
+\, ↔ '--..-- ' ;
+\: ↔ '---... ' ;
+
+፣ → '.-.-.- ' ;
+፥ → '.-.-.- ' ;
+፨ → '...... ' ;
+፠ → '...... ' ;
+፧ → '..--..' ;
+፡ ↔ '-.-.-. ' ;
+\; → '-.-.-. ' ;
+‹ → '.----. ' ;
+› → '.----. ' ;
+\' ↔ '.----. ' ;
+« → '.-..-. ' ;
+» → '.-..-. ' ;
+
+
+########################################################################
+# Start of Numeric Transformations
+########################################################################
+፳ → | 20 ; # ETHIOPIC NUMBER TWENTY
+፴ → | 30 ; # ETHIOPIC NUMBER THIRTY
+፵ → | 40 ; # ETHIOPIC NUMBER FORTY
+፶ → | 50 ; # ETHIOPIC NUMBER FIFTY
+፷ → | 60 ; # ETHIOPIC NUMBER SIXTY
+፸ → | 70 ; # ETHIOPIC NUMBER SEVENTY
+፹ → | 80 ; # ETHIOPIC NUMBER EIGHTY
+፺ → | 90 ; # ETHIOPIC NUMBER NINETY
+፪ → |  2 ; # ETHIOPIC DIGIT TWO
+፫ → |  3 ; # ETHIOPIC DIGIT THREE
+፬ → |  4 ; # ETHIOPIC DIGIT FOUR
+፭ → |  5 ; # ETHIOPIC DIGIT FIVE
+፮ → |  6 ; # ETHIOPIC DIGIT SIX
+፯ → |  7 ; # ETHIOPIC DIGIT SEVEN
+፰ → |  8 ; # ETHIOPIC DIGIT EIGHT
+፱ → |  9 ; # ETHIOPIC DIGIT NINE
+፼ → | 10000 ; # ETHIOPIC NUMBER TEN HUNDRED
+፻ → | 100 ; # ETHIOPIC NUMBER HUNDRED 
+፲ → |  10 ; # ETHIOPIC NUMBER TEN
+፩ → |   1 ; # ETHIOPIC DIGIT ONE
+
+1 ↔ '.---- ' ;
+2 ↔ '..--- ' ;
+3 ↔ '...-- ' ;
+4 ↔ '....- ' ;
+5 ↔ '..... ' ;
+6 ↔ '-.... ' ;
+7 ↔ '--... ' ;
+8 ↔ '---.. ' ;
+9 ↔ '----. ' ; 
+0 ↔ '----- ' ; 
+
+#
+# TODO: Seek a better way to handle this. The conversion adds a space to the end of every sequence to terminate it.
+#       The CLDR "TestTransforms" unit test will strip off the space at the end of a line in the am-Latn-t-am-Ethi-m0-morse-code.txt
+#       test file thus breaking the comparison with generated words.  The following will strip off the space at the end of
+#       a converted string to fascilitate compatibility with the unit tests.
```

Comment:
it makes some sense to not generate an extra space. However this will make  `73` -> `--... ...--` whereas `7`+`3` = `--......--`

Reply:
Converted letters would run together too.  For example "at" is read clearly as ".- -" . With the " " added after ".-"  the conversion would be ".--" which would be misinterpreted as "w" .  The issue is the space between characters, but not between words.  At the end of a word though, the last character still gets the " " which is now superfluous since a word boundary token is naturally expected.

Thinking about it again now though... "/" is the normal word boundary in ASCII form (the conversion of " "), which this transliterator is not supporting.  I should add it if everything else looks ok.

Reply:
@srl295 , the mapping of  `73` -> `--... ...--` , might be OK.  The https://morsedecoder.com/ produces this output (taking a leap of faith here to assume the service is valid).